src/style.css

```diff
@@ -7,10 +7,17 @@
 }
 
 body {
+  width: 97vw;
+}
+
```

Comment:
![image](https://user-images.githubusercontent.com/43698511/133697472-63ba5316-e60f-43bf-8b42-0b253ee65c84.png)

- [ ] As you can see in the picture the horizontal scroll bar is showing for mobile view under 432px. Kindly fix the issue.
![image](https://user-images.githubusercontent.com/43698511/133697602-e700fe1b-5eb5-4329-88e0-549bf5364c1f.png)
original design:
![image](https://user-images.githubusercontent.com/43698511/133697631-554cd974-8795-4eaa-82c0-36fb1f73a485.png)

- [ ] As you can see in the picture your project looks a bit different from the original design in mobile view. Kindly fix the issue.